websocket/session_store.go

```diff
@@ -0,0 +1,78 @@
+package websocket
+
+import (
+	"github.com/cortezaproject/corteza-server/pkg/id"
+	"sync"
+)
+
+var store *Store
+
+type (
+	Store struct {
```

Comment:
there's no need for this to be published, right? Plus... might be best if we find a better name for this, since this was originally implemented, we introduced "Store" for something else.

Even `sessions` (so, plural) is good enough.